website/src/css/_config.scss

```diff
@@ -0,0 +1,348 @@
+  --#{$prefix}-y5a: #f7ba2912;
+  --#{$prefix}-b5a: #3e77e412;
+  --#{$prefix}-u5a: #7633e512;
+  --#{$prefix}-e5a: #3ebf7212;
+
+  --#{$prefix}-a0: #12161bff;
+  --#{$prefix}-a1: #babbbdff;
+  --#{$prefix}-a2: #e0e1e1ff;
+  --#{$prefix}-a3: #f1f1f1ff;
+  --#{$prefix}-a4: #fafafaff;
+  --#{$prefix}-a5: #ffffffff;
+
+  --#{$prefix}-p0: #5c25afff;
+  --#{$prefix}-p1: #8157c1ff;
+  --#{$prefix}-p2: #9876cdff;
+  --#{$prefix}-p3: #a789d4ff;
+  --#{$prefix}-p4: #b9a1ddff;
+  --#{$prefix}-p5: #d3c4e9ff;
+}
+
+@mixin ifm-colors($prefix) {
+  --#{$prefix}-color-primary: #048fc0;
+  --#{$prefix}-color-primary-dark: #253282;
+  --#{$prefix}-color-primary-darker: #161360;
+  --#{$prefix}-color-primary-darkest: #150a51;
+  --#{$prefix}-color-primary-light: #6c43a3;
+  --#{$prefix}-color-primary-lighter: #8859c9;
+  --#{$prefix}-color-primary-lightest: #9f6de5;
+  --#{$prefix}-code-font-size: 95%;
+  --#{$prefix}-highlighted-code-line-bg: #d9e5f6;
+}
+
+@mixin detox-colors-dark($prefix) {
+  --#{$prefix}-highlighted-code-line-bg: #414458;
+  --ifm-background-color: #12161b;
+
+  --#{$prefix}-color-a0: #ffffff;
+  --#{$prefix}-color-a1: #12161b;
+  --#{$prefix}-color-a2: #171b20;
+  --#{$prefix}-color-a3: #202429;
+  --#{$prefix}-color-a4: #313439;
+  --#{$prefix}-color-a5: #575a5d;
+}
+
+:root {
+  @include detox-colors('detox-color');
+  @include ifm-colors('ifm');
+
+  &[data-theme='dark'] {
+    @include detox-colors-dark('detox-color');
+  }
+}
+
+// Fonts
+
+@mixin h1-80 {
+  font-family: 'Nunito Sans';
+  font-size: 80px;
+  font-weight: 400;
+}
+
+@mixin h1-96 {
+  font-family: 'Nunito Sans';
+  font-size: 96px;
+  font-weight: 400;
+}
+
+@mixin h2-58 {
+  font-family: 'Nunito Sans';
+  font-size: 58px;
+  font-weight: 700;
+}
+
+@mixin h2-70 {
+  font-family: 'Nunito Sans';
+  font-size: 70px;
+  font-weight: 700;
+}
+
+@mixin h3-40 {
+  font-family: 'Nunito Sans';
+  font-size: 40px;
+  font-weight: 600;
+}
+
+@mixin h3-50 {
+  font-family: 'Nunito Sans';
+  font-size: 50px;
+  font-weight: 600;
+}
+
+@mixin h4-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 700;
+}
+
+@mixin h4-40 {
+  font-family: 'Nunito Sans';
+  font-size: 40px;
+  font-weight: 700;
+}
+
+@mixin h5-24 {
+  font-family: 'Nunito Sans';
+  font-size: 24px;
+  font-weight: 400;
+}
+
+@mixin h5-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 400;
+}
+
+@mixin t1-20 {
+  font-family: 'Nunito Sans';
+  font-size: 20px;
+  font-weight: 400;
+}
+
+@mixin t1-30 {
+  font-family: 'Nunito Sans';
+  font-size: 30px;
+  font-weight: 400;
+}
+
+@mixin t2-16 {
+  font-family: 'Nunito Sans';
+  font-size: 16px;
+  font-weight: 400;
+}
+
+@mixin t2-26 {
+  font-family: 'Nunito Sans';
+  font-size: 26px;
+  font-weight: 400;
+}
+
+@mixin t3-13 {
+  font-family: 'Nunito Sans';
+  font-size: 13px;
+  font-weight: 400;
+}
+
+@mixin t3-20 {
+  font-family: 'Nunito Sans';
+  font-size: 20px;
+  font-weight: 400;
+}
+
+// Showcase buttons
```

Comment:
Isn't that part of the Showcase? Is that the card's buttons? This needs to be under showcase-specific file

Reply:
Done!